plugins/rikki/loungestatistics/classes/statistics/Statistics.php

```diff
@@ -0,0 +1,166 @@
+<?php namespace Rikki\LoungeStatistics\classes\Statistics;
+
+use Rikki\Heroeslounge\Models\Hero;
+use Rikki\Heroeslounge\Models\Map;
+
+use October\Rain\Support\Collection;
+
+class Statistics
+{
+    /*
+        Current supported types are:
+            - division -> Division api endpoint.
+            - sloth -> sloth hero statistics endpoint.
+            - slothAll -> sloth statistics component.
+
+        Our current types of rawData are:
+        - Matches
+        - gameParticipations        
+    */
+    public static function calculateHeroStatistics($type, $rawData, $season)
```

Comment:
Please make sure that these parameters always have the same type - $rawData can be either matches or gameParticipations. In any case, please write in the comment which $type supports which $rawData (or which parameters).